fbbt/src/ontologies/fbbt-edit.obo

```diff
@@ -63575,6 +63574,260 @@ intersection_of: FBbt:00005106 ! neuron
+is_a: FBbt:00007701 ! longitudinal subdivision of central nervous system
+relationship: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T14:07:31Z
+
+[Term]
+id: FBbt:00007709
+name: central subdivision of ventral nerve cord
+def: "A centrally located longitudinal subdivision of the larval ventral nerve cord spanning the gnathal, thoracic and abdominal segments to abdominal segment 8. This compartment surrounds the CITd (centro intermediate tract dorsal), `CI1` [FBbt_00005905]) and CITv fascicles. (CITv encompasses `CI2`[FBbt_00005906] and `CI3`[FBbt_00005907])" [FB:dos] {comment="PMID:28730682"}
+is_a: FBbt:00007701 ! longitudinal subdivision of central nervous system
+relationship: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-01-22T14:09:26Z
+
+[Term]
+id: FBbt:00007710
+name: ventral nerve cord synaptic neuropil subdomain
+intersection_of: FBbt:00040006 ! synaptic neuropil subdomain
+intersection_of: part_of FBbt:00001102 ! ventral nerve cord
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-02T18:33:08Z
+
+[Term]
+id: FBbt:00007711
+name: larval T1 leg neuropil
+def: "A protruding, spherical neuropil on the dorso-lateral side of the segment T1 of the ventral nerve cord, embedded into the dorso-lateral neuropil domains of T1." [FBC:dw, PMID:28730682]
+is_a: FBbt:00007700 ! embryonic/larval leg neuropil
+relationship: part_of FBbt:00001743 ! larval prothoracic segment
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-14T10:06:29Z
+
+[Term]
+id: FBbt:00007712
+name: larval T2 leg neuropil
+def: "A protruding, spherical neuropil on the dorso-lateral side of the segment T2 of the ventral nerve cord, embedded into the dorso-lateral and centro-lateral neuropil domains of T2." [FBC:dw, PMID:28730682]
+is_a: FBbt:00007700 ! embryonic/larval leg neuropil
+relationship: part_of FBbt:00001743 ! larval prothoracic segment
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-14T10:06:29Z
+
+[Term]
+id: FBbt:00007713
+name: larval T3 leg neuropil
+def: "A protruding, spherical neuropil on the dorso-lateral side of the segment T3 of the ventral nerve cord, embedded into the dorso-lateral and centro-lateral neuropil domains of T3." [FBC:dw, PMID:28730682]
+is_a: FBbt:00007700 ! embryonic/larval leg neuropil
+relationship: part_of FBbt:00001743 ! larval prothoracic segment
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-14T10:11:13Z
+
```

Comment:
A9 (below): The separations are such that they follow those of the rest of the VNC ventro-centro-dorsal, thus the following text should be added:

Dorsal domain: "The axial plane placed just dorsally of the intermediate commissures and CITd throughout the VNC separates the central from the dorsal domain."

Ventral domain: "The virtual axial plane intersecting the CITv and the VMT throughout the VNC separates the central from the ventral domain."

Central domain: "The virtual axial plane intersecting the CITv and the VMT throughout the VNC separates the central from the ventral domain.  The axial plane placed just dorsally of the intermediate commissures and CITd throughout the VNC separates the central from the dorsal domain."